cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java

```diff
@@ -68,12 +69,13 @@
   private List<BlockingQueue<SendLogRequest>> nodeLogQueues =
       new ArrayList<>();
   private ExecutorService executorService;
```

Comment:
LGTM!
only one question, why only change the _serializationService_ to use google's ThreadFactoryBuilder,
Why not change the _executorService_? Is there any special thinking?

Reply:
Sure I can do this, but I was focusing on serializationService only and ignored executorService. I will modify it too.

Reply:
I manually set the thread name in DispatcherThread, so the receiver's name can be included in the thread name, which is not convenient when using ThreadFactoryBuilder.